requirements/base.txt

```diff
@@ -7,16 +7,17 @@ kornia==0.5.6
 lxml==4.6.5
 matplotlib==3.4.3
 networkx~=2.5
-numpy~=1.19.5
+# nncf==2.1.0
+numpy>=1.22.3
 omegaconf==2.1.1
+pandas==1.4.1
 opencv-python>=4.5.3.56
 opencv-contrib-python==4.5.5.62
-pandas~=1.1.5
 pillow==9.0.0
-pytorch-lightning==1.5.9
-torch==1.8.1
-torchvision==0.9.1
-torchtext==0.9.1
+pytorch-lightning==1.6.0
```

Comment:
pytorch-lightning 1.6.0 modified some of the annotations under the hood. CI is failing due to these changes. 
[CI-PR-158.log](https://github.com/openvinotoolkit/anomalib/files/8427463/CI-PR-158.log)